compute/instances/concepts.mdx

```diff
@@ -125,11 +125,15 @@ An InstantApp is an image with a preinstalled application. By choosing an Instan
 
 ## IP mobility
 
-The legacy public network is provided through a network component managing a carrier-grade NAT with private IP addresses.
-When an Instance is started, it is assigned a private IP address that is associated with the underlying node hosting the Instance.
-The carrier-grade NAT establishes a one-to-one mapping between the public IP and the private IP assigned to the Instance during provisioning. Consequently, if the underlying node changes, the private IP address will also change.
+The legacy public network is provided through carrier-grade NAT infrastructure.
+When an Instance is started, it is assigned a NAT IP address that is associated with the underlying node hosting the Instance.
+The carrier-grade NAT establishes a one-to-one mapping between the public IP and the NAT IP assigned to the Instance during provisioning. Consequently, if the underlying node changes, the NAT IP address will also change.
 
-With IP mobility, there are no private IPs anymore. The public IP is entirely routed to the Instance. The main benefit is that your underlying IP no longer changes when you stop and start your Instance or if it gets provisioned on another hypervisor. But this also means that if you do not attach a public IP address to your Instance, then it will not have any network access.
+With IP mobility, there are no NAT IPs anymore. The public IP is entirely routed to the Instance. The main benefit is that your underlying IP no longer changes when you stop and start your Instance or if it gets provisioned on another hypervisor. But this also means that if you do not attach a public IP address to your Instance, then it will not have any network access.
```

Comment:
```suggestion
With IP mobility, NAT IPs are no longer available. The public IP is entirely routed to the Instance. The main benefit is that your underlying IP no longer changes when you stop and start your Instance or if it gets provisioned on another hypervisor. But this also means that if you do not attach a public IP address to your Instance, then it will not have any network access.
```